app/assets/javascripts/application.js

```diff
@@ -12,5 +12,4 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
```

Comment:
remove turbolinks

Reply:
The highlighted red line indicates it's been removed already.